dashboard/src/App.js

```diff
@@ -96,11 +96,11 @@ const App = () => {
                       path={APP_ROUTES.ANALYSIS}
                       element={<ComingSoonPage />}
                     />
-                    <Route
-                      path={APP_ROUTES.VISUALIZATION}
-                      element={<ComparisonComponent />}
-                    />
                   </Route>
+                  <Route
+                    path={APP_ROUTES.VISUALIZATION}
+                    element={<ComparisonComponent />}
+                  />
```

Comment:
It'd be good to get this change in, as well. (This could affect the UI if you do the "toggle" for Results visibility since "show mine" is meaningless if not logged in, and there's no difference between "show all" and "show public" in that case. I think this would just mean suppressing the "toggle" if not logged in, since no special filters are necessary for the only meaningful view.)